google/resource_container_cluster.go

```diff
@@ -66,6 +66,11 @@ func resourceContainerCluster() *schema.Resource {
 							Required: true,
 							ForceNew: true,
 						},
+						"issue_client_certificate": {
+							Type:             schema.TypeBool,
+							Optional:         true,
```

Comment:
I think this needs to be `ForceNew` as well.

Reply:
Done